tools/sesman/tests/test_targetaccuratefilter.py

```diff
@@ -387,6 +361,6 @@ def test_is_filterable_with_parsing_error_on_filter_patterns_6(self) :
             "service" : "my_service"}
 
         self.assertRaises(ParsingError,
-                          targetaccuratefilter.is_filterable,
+                          taf.is_filterable,
                           FILTER_PATTERNS,
                           TARGET_FIELD_DICT)
```

Comment:
The tests are too scattered, it is difficult to read them and get a general overview of what is being tested.

```py
def test_is_filterable_with_parsing_error_on_filter_patterns(self):
    def assertParsingError(patt):
        with self.subTest(patt=patt):
            self.assertRaises(ParsingError, taf.is_filterable, patt, {})

    assertParsingError("pat1")
    assertParsingError("pat2")
    assertParsingError("pat3")
    ....
```

Same for all other tests.